acquisitions/src/main/resources/samples/mod-gobi/po-listed-electronic-monograph.xml

```diff
@@ -49,7 +49,7 @@
         <FundCode>xnath</FundCode>
         <OrderNotes />
         <OtherLocalId>.o22310460</OtherLocalId>
-        <Location>NATH</Location>
+        <Location>LOC4</Location>
```

Comment:
these locations belong to diku and not found from search/consortium/locations so created new locations four and using them